source/dump.f90

```diff
@@ -186,27 +186,26 @@ subroutine dump_lines(n,i,ix)
 
   use mod_common_track
 
-  implicit none
```

Comment:
Is this simply redundant because it's in the module already?

Reply:
Yes